src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -93,6 +94,7 @@ Profile::Profile(const std::optional<GUID>& guid) :
     _selectionBackground{},
     _colorTable{},
     _tabTitle{},
+    _suppressApplicationTitle{},

```

Comment:
```suggestion
    _suppressApplicationTitle{ false },
```

Reply:
Isn't this only if it's set false as default?